docs/pages/connect-your-client/gui-clients.mdx

```diff
@@ -317,8 +344,16 @@ Provide your Redis username as `Username` and password as `Password`.
 
 ![Redis Insight Configuration](../../img/database-access/guides/redis/redisinsight-add-config.png)
 
-Next, check the `Use TLS` and `Verify TLS Certificates` boxes and copy the CA certificate returned by `tsh proxy db`.
-Copy the private key and certificate to corresponding fields.
+Next, check the `Use TLS` and `Verify TLS Certificates` boxes. Copy the files at
+the paths returned by the `tsh proxy db` command you ran earlier and paste them
+into their corresponding fields. See the table below for the Redis Insight
+fields that correspond to each path:
+
+|Field|Path|
+|---|---|
+|Client Certificate|`cert_file`|
+|Private Key|`key_file`|
+|CA Certificate|`ca_file`|
```

Comment:
I wonder if it'd make sense to reflect the order of fields in the UI here? So it'd be CA cert, client cert and private key.